<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title>Matthias GrÃ¶schel</title>
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="1504.84">
  <style type="text/css">
    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000; min-height: 14.0px}
    p.p2 {margin: 0.0px 0.0px 16.1px 0.0px; text-align: center; line-height: 28.0px; font: 24.0px Times; color: #000000; -webkit-text-stroke: #000000}
    p.p3 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #24292e; -webkit-text-stroke: #24292e}
    p.p4 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #24292e; -webkit-text-stroke: #24292e; background-color: #ffffff}
    p.p5 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000}
    p.p6 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #0000ee; -webkit-text-stroke: #0000ee}
    p.p7 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000; min-height: 14.0px}
    p.p8 {margin: 0.0px 0.0px 12.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000; min-height: 14.0px}
    p.p9 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px Times}
    p.p10 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px Times; min-height: 13.0px}
    p.p11 {margin: 0.3px 0.0px 0.0px 0.0px; text-align: center; font: 9.5px Times; min-height: 11.0px}
    p.p12 {margin: 0.0px 0.0px 0.0px 7.0px; text-align: center; font: 10.0px Times}
    p.p13 {margin: 0.0px 0.0px 0.0px 5.0px; text-align: center; font: 10.0px Times}
    p.p15 {margin: 0.0px 0.0px 0.0px 25.0px; text-align: center; font: 10.0px Times; min-height: 13.0px}
    p.p19 {margin: 0.6px 0.0px 0.0px 0.0px; text-align: center; font: 9.5px Times; min-height: 11.0px}
    p.p21 {margin: 0.1px 0.0px 0.0px 0.0px; text-align: center; font: 10.5px Times; min-height: 13.0px}
    p.p22 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px 'Times New Roman'}
    p.p23 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px 'Times New Roman'; min-height: 11.0px}
    p.p24 {margin: 0.0px 0.0px 0.0px 72.0px; text-align: center; font: 10.0px 'Times New Roman'}
    p.p25 {margin: 0.0px 0.0px 0.0px 72.0px; text-align: center; font: 10.0px 'Times New Roman'; color: #0079cd}
    p.p26 {margin: 0.0px 0.0px 0.0px 36.0px; text-align: center; text-indent: 36.0px; font: 10.0px 'Times New Roman'}
    p.p27 {margin: 0.0px 0.0px 0.0px 72.0px; text-align: center; text-indent: -72.0px; font: 10.0px 'Times New Roman'}
    p.p28 {margin: 0.0px 0.0px 0.0px 25.1px; text-align: center; font: 10.0px 'Times New Roman'}
    p.p29 {margin: 0.0px 0.0px 0.0px 72.0px; text-align: center; font: 10.0px 'Times New Roman'; color: #2f2f2f}
    p.p30 {margin: 0.0px 0.0px 12.0px 0.0px; text-align: center; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000}
    li.li9 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px Times}
    li.li14 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px Times; color: #2f2f2f}
    li.li16 {margin: 0.1px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px Times}
    li.li17 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 12.1px; font: 10.0px Times; color: #2f2f2f}
    li.li18 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; line-height: 11.8px; font: 10.0px Times}
    li.li20 {margin: 0.2px 0.0px 0.0px 0.0px; text-align: center; font: 10.0px Times}
    span.s1 {font-kerning: none}
    span.s2 {font-kerning: none; background-color: #ffffff}
    span.s3 {text-decoration: underline ; font-kerning: none}
    span.s4 {font-kerning: none; color: #000000; -webkit-text-stroke: 0px #000000}
    span.s5 {text-decoration: underline ; font-kerning: none; color: #0000ee; -webkit-text-stroke: 0px #0000ee}
    span.s6 {font: 12.0px Menlo; font-kerning: none}
    span.s7 {color: #353535}
    span.s8 {text-decoration: underline}
    span.s9 {text-decoration: underline ; letter-spacing: -0.3px}
    span.s10 {letter-spacing: -1.2px}
    span.s11 {color: #2f2f2f}
    span.s12 {letter-spacing: -0.1px}
    span.s13 {font: 12.0px Times}
    span.s14 {letter-spacing: -1.0px}
    span.s15 {color: #000000}
    span.s16 {letter-spacing: -1.8px; color: #000000}
    span.s17 {letter-spacing: -0.4px}
    span.s18 {letter-spacing: -0.7px}
    span.s19 {letter-spacing: -0.2px}
    span.s20 {letter-spacing: 0.1px; color: #000000}
    span.s21 {letter-spacing: -1.2px; color: #000000}
    span.s22 {letter-spacing: -0.3px}
    span.s23 {letter-spacing: 0.1px}
    span.s24 {letter-spacing: -1.1px}
    span.s25 {text-decoration: underline ; letter-spacing: 0.1px}
    span.s26 {text-decoration: underline ; letter-spacing: -0.1px}
    span.s27 {letter-spacing: -0.9px}
    span.s28 {letter-spacing: 0.8px}
    span.s29 {letter-spacing: -1.3px}
    span.s30 {letter-spacing: 0.7px}
    span.Apple-tab-span {white-space:pre}
    ul.ul1 {list-style-type: disc}
  </style>
</head>
<body>
<p class="p1"><span class="s1"></span><br></p>
<p class="p1"><span class="s1"></span><br></p>
<p class="p1"><span class="s1"><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1"><b>Bana Alamad</b></span></p>
<p class="p3"><span class="s2">MPhil in Genomics Medicine Candidate<span class="Apple-converted-space"> </span></span></p>
<p class="p4"><span class="s1">Department of Medical Genetics</span></p>
<p class="p3"><span class="s2">University of Cambridge</span></p>
<p class="p5"><span class="s1">Cambridge, United Kingdom</span></p>
<p class="p1"><span class="s1"></span><br></p>
<p class="p6"><span class="s3"><a href="mailto:matthias_groeschel@hms.harvard.edu">matthias_groeschel@hms.harvard.edu</a></span><span class="s4"><span class="Apple-converted-space"> </span></span></p>
<p class="p1"><span class="s1"></span><br></p>
<p class="p6"><span class="s4">[<a href="file:///Users/banahal-amad/Desktop/cv.pdf"><span class="s5">CV (will be updated soon)</span></a>]<span class="Apple-converted-space"> </span></span></p>
<p class="p1"><span class="s1"></span><br></p>
<p class="p5"><span class="s1">About</span></p>
<p class="p4"><span class="s6"><span class="Apple-converted-space"> </span></span><span class="s1">I graduated with a degree in biology from New York university<span class="Apple-converted-space"> </span></span></p>
<p class="p7"><span class="s1"></span><br></p>
<p class="p5"><span class="s1">Research</span></p>
<p class="p8"><span class="s3"></span><br></p>
<p class="p9"><span class="Apple-converted-space">  </span>The Farhat Lab Harvard Medical School, <span class="s7">Department of Biomedical Informatics</span>, USA<span class="Apple-tab-span">	</span><span class="Apple-converted-space">              </span><span class="s8">June 2020 – Present<span class="Apple-converted-space"> </span></span></p>
<p class="p9"><span class="Apple-converted-space">  </span>Research Assistant (Analytical)</p>
<ul class="ul1">
  <li class="li9">•<span class="Apple-tab-span">	</span>Single-cell and bulk RNA-seq analysis to identify immune signatures in response to the vaccination of rhesus macaques with bacille Calmette–Guérin (BCG) vaccine against Mycobacterium tuberculosis (Mtb).</li>
  <li class="li9">•<span class="Apple-tab-span">	</span>Data trimming, annotation, quality control, and alignment. HPC computing, Python, Snakemake.<span class="Apple-converted-space"> </span></li>
  <li class="li9">•<span class="Apple-tab-span">	</span>Presented at journal clubs, lab meetings, and attended Harvard Medical School departmental events. <span class="Apple-converted-space"> </span></li>
</ul>
<p class="p10"><br></p>
<p class="p9"><span class="Apple-converted-space">  </span>The Clinical AI Research Lab, Engineering Division NYUAD, UAE<span class="Apple-tab-span">	</span> <span class="Apple-converted-space">   <span class="Apple-tab-span">	</span>                             </span><span class="s8">July 2020 – November 2020</span></p>
<p class="p9"><span class="Apple-converted-space">  </span>Research Assistant (Analytical)</p>
<ul class="ul1">
  <li class="li9">•<span class="Apple-tab-span">	</span><span class="s8">Co-authored publication (submitted, under review) </span>in The Lancet Digital Health journal titled: “Clinical prediction system of complications among COVID-19 patients: a development and validation retrospective multicentre study”.</li>
  <li class="li9">•<span class="Apple-tab-span">	</span><span class="s8">Co-authored publication (to be submitted in December 2020) </span>in AAAI symposium 2021 titled: “Prediction of acute respiratory distress syndrome ARDS using free-text chest radiology reports for COVID-19 patients”.</li>
  <li class="li9">•<span class="Apple-tab-span">	</span>Presented an automated machine learning system that achieves strong performance to predict 7 COVID-19 complications. <span class="Apple-converted-space"> </span></li>
  <li class="li9">•<span class="Apple-tab-span">	</span>Clinical interpretation of model outcomes for patient deterioration, in collaboration with 4 physicians.<span class="Apple-converted-space"> </span></li>
  <li class="li9">•<span class="Apple-tab-span">	</span>Created high quality figures, wrote and edited publications.<span class="Apple-converted-space"> </span></li>
</ul>
<p class="p11"><br></p>
<p class="p12">Idaghdour Environmental Genetics Lab, Science Division NYUAD, UAE<span class="Apple-tab-span">	</span><span class="Apple-converted-space">  </span><span class="s8">September 2018 - May</span><span class="s9"> </span><span class="s8">2020</span></p>
<p class="p13">Undergraduate Thesis Project</p>
<ul class="ul1">
  <li class="li14"><span class="Apple-tab-span">	</span>ATAC-Seq Profiling of Peripheral Blood Mononuclear Cells in response to P. falciparum infection in Burkina Faso.</li>
  <li class="li14"><span class="Apple-tab-span">	</span>Wrote an NIH-format research proposal, presented it, defended it to a scientific committee, and received<span class="s10"> </span>funding.</li>
  <li class="li9"><span class="s11"><span class="Apple-tab-span">	</span>Generated the first ATAC-seq dataset in malaria, </span>uncovered cell-type specific temporal chromatin accessibility changes in response to infection, integrated findings with single-cell RNA-seq data to analyze patient’s immune response.<span class="Apple-converted-space"> </span></li>
  <li class="li9"><span class="Apple-tab-span">	</span><span class="s8">First author publication (to be submitted in December 2020)</span> in PLOS Genetics journal (impact<span class="s12"> </span>factor<span class="s12"> </span>5.0).<span class="Apple-converted-space"> </span></li>
  <li class="li9"><span class="Apple-tab-span">	</span><span class="s8">Accepted Abstract and poster presentation </span>in “Single-cell Biology 2020” Welcome Genome Campus, UK.</li>
  <li class="li9"><span class="Apple-tab-span">	</span><span class="s8">Accepted Abstract and poster presentation </span>in “Genome Systems X” Center for Genomics and Systems Biology Symposium, UAE.<span class="s13"><span class="Apple-converted-space"> </span></span></li>
</ul>
<p class="p15"><br></p>
<p class="p13">Research Assistant (Experimental)</p>
<ul class="ul1">
  <li class="li16"><span class="Apple-tab-span">	</span>Handled peripheral blood mononuclear cells (PBMCs), FACS, MACS, T cell, B cell, and Monocyte<span class="s14"> </span>isolation.</li>
  <li class="li17"><span class="s15"><span class="Apple-tab-span">	</span>Performed </span>gel electrophoresis, DNA/RNA extraction, bioanalyzer analysis, NGS <span class="s15">single-cell transcriptional</span><span class="s16"> </span><span class="s15">profiling.</span></li>
  <li class="li14"><span class="Apple-tab-span">	</span>Performed<span class="s17"> </span>bulk<span class="s12"> </span>and<span class="s12"> </span>single-cell<span class="s17"> </span>chromatin<span class="s12"> </span>accessibility<span class="s12"> </span>mapping<span class="s18"> </span>NGS<span class="s19"> </span>epigenetic<span class="s12"> </span>profiling<span class="s12"> </span>(<span class="s15">10X</span><span class="s20"> </span><span class="s15">Genomics</span><span class="s21"> </span><span class="s15">Chromium</span> ATAC-seq)</li>
</ul>
<p class="p12">Research Assistant (Analytical)</p>
<ul class="ul1">
  <li class="li9"><span class="Apple-tab-span">	</span>GWAS analysis, whole exome data from 95 individuals to map disease-causing variants using JMP<span class="s22"> </span>Genomics.</li>
  <li class="li9"><span class="Apple-tab-span">	</span>Differential gene expression analysis on bulk RNA-seq data, cross-correlation analysis (gene expression and methylation data) using R package DESeq2 and JMP<span class="s23"> </span>Genomics.</li>
  <li class="li18"><span class="Apple-tab-span">	</span>ATAC-seq data analysis using esATAC and<span class="s22"> </span>Homer.</li>
  <li class="li16"><span class="Apple-tab-span">	</span>T-cell receptor (TCR) profiling analysis and plotting using JMP, R, and ImmunoSEQ<span class="s22"> </span>Analyzer.</li>
  <li class="li9"><span class="Apple-tab-span">	</span>R Bioconductor, R DESeq2, JMP genomics, <span class="s22">IGV </span>(data visualization), statistical analysis, ImmunoSEQ Analyzer, BLAST,<span class="s12"> </span>Panther.</li>
</ul>
<p class="p19"><br></p>
<p class="p12">Toseland Laboratory University <span class="s12">of </span>Kent, Kent<span class="s24"> </span>UK<span class="Apple-tab-span">	</span><span class="s8">June 2018- December</span><span class="s25"> </span><span class="s8">2018</span></p>
<p class="p12">Research Assistant: Regulation of Myosin VI Cellular Function</p>
<ul class="ul1">
  <li class="li16"><span class="Apple-tab-span">	</span><span class="s8">Co-authored</span><span class="s26"> </span><span class="s8">publication (published)</span><span class="s12"> </span>in<span class="s12"> </span>the Journal<span class="s12"> </span>of<span class="s12"> </span>Biological<span class="s12"> </span>Chemistry<span class="s19"> </span>(impact<span class="s12"> </span>factor<span class="s12"> </span>4.1)<span class="s12"> </span>titled:<span class="s12"> </span>“Competition<span class="s12"> </span>between<span class="s27"> </span>two high- and low-affinity protein-binding sites in myosin VI controls its cellular<span class="s22"> </span>function”.</li>
  <li class="li18"><span class="Apple-tab-span">	</span>HELA and MCF7 cell culture, cell line maintenance, and<span class="s18"> </span>transfection.</li>
  <li class="li9"><span class="Apple-tab-span">	</span>In vitro Immunofluorescence staining, Immunoblot Analysis, and Fluorescence<span class="s28"> </span>Imaging.</li>
  <li class="li20"><span class="Apple-tab-span">	</span>Binding partner (Dab2 and CALCOCO2/NDP52) truncation, protein<span class="s22"> </span>recombination.</li>
  <li class="li9"><span class="Apple-tab-span">	</span>Uncovered Myosin VI’s mechanism for regulating its transcriptional activity, concluded that the loss of DAB2 enhances myosin VI– mediated during cancer<span class="s12"> </span>onset.</li>
</ul>
<p class="p21"><br></p>
<p class="p10"><br></p>
<p class="p9"><span class="Apple-converted-space">   </span>Columbia University’s Department <span class="s12">of </span>Genetics and Development,<span class="s29"> </span>USA<span class="Apple-tab-span">	</span><span class="s8">April</span><span class="s9"> </span><span class="s8">2019</span></p>
<p class="p12">Genetic Counselling Shadow</p>
<ul class="ul1">
  <li class="li16"><span class="Apple-tab-span">	</span>Shadowed a genetic counsellor, attended departmental meetings, and engaged in discussions analyzing patient’s genomic data with physicians.</li>
  <li class="li16"><span class="Apple-tab-span">	</span>Attended the Human Genetics NYC conference and participated in a workshop discussing the legal and<span class="s24"> </span>bioethical implications of the return of genetic results to<span class="s30"> </span>patients.</li>
</ul>
<p class="p8"><span class="s1"></span><br></p>
<p class="p22"><span class="s8">Awards:</span></p>
<p class="p23"><br></p>
<p class="p22">2020 <span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><b>Bseisu-Cambridge Scholarship</b><span class="Apple-converted-space"> </span></p>
<p class="p24">First woman to be awarded the Bseisu-Cambridge scholarship since the foundation was established in 2005</p>
<p class="p24">University of Cambridge, UK</p>
<p class="p25"><span class="s15">Full tuition and living expenses covered <br>
Article in News: <a href="https://gulfnews.com/uae/education/abu-dhabi-graduate-becomes-first-to-receive-bseisu-cambridge-scholarship-1.73084009"><span class="s8">https://gulfnews.com/uae/education/abu-dhabi-graduate-becomes-first-to-receive-bseisu-cambridge-scholarship-1.73084009</span></a></span></p>
<p class="p23"><br></p>
<p class="p22">2016- 2020 <span class="Apple-tab-span">	</span><b>Full financial Aid Schoalrship</b><span class="Apple-converted-space"> </span></p>
<p class="p22"><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span>New York University, UAE</p>
<p class="p22"><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span>Full tuition and living expenses covered</p>
<p class="p26">$85,000 per year<span class="Apple-converted-space"> </span></p>
<p class="p23"><br></p>
<p class="p22">2016 <span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><b>Academic Excellence Golden award</b> (given to 3 out of 40 graduates)</p>
<p class="p22"><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><b>Leadership Golden award (</b>given to 2 out of 40 graduates)</p>
<p class="p26">Mashrek International School, Jordan<span class="Apple-converted-space"> </span></p>
<p class="p23"><span class="Apple-converted-space"> </span></p>
<p class="p22"><span class="s8">Publications:<span class="Apple-converted-space"> </span></span></p>
<p class="p23"><br></p>
<p class="p27">2020 <span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><b>Co-authored publication, </b>The Lancet Digital Health journal</p>
<p class="p27"><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span>“Clinical prediction system of complications among COVID-19 patients: a development and validation retrospective multicentre study”.</p>
<p class="p22"><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><span class="s8"><b><i>submitted, under review</i></b></span></p>
<p class="p23"><i></i><br></p>
<p class="p28"><b><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span>Co-authored publication, </b>AAAI symposium 2021</p>
<p class="p24">“Prediction of acute respiratory distress syndrome ARDS using free-text chest radiology reports for COVID-19 patients”.</p>
<p class="p26"><span class="s8"><b><i>to be submitted in November 2020</i></b></span></p>
<p class="p23"><i></i><br></p>
<p class="p22"><i><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span></i><b>First author publication, </b>PLOS Genetics journal</p>
<p class="p29">“ATAC-Seq Profiling of Peripheral Blood Mononuclear Cells in response to <i>P. falciparum</i> infection in Burkina Faso”.</p>
<p class="p26"><span class="s8"><b><i>to be submitted in December 2020</i></b></span></p>
<p class="p23"><br></p>
<p class="p22">2019 <span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><b>Co-authored</b><span class="s12"><b> </b></span><b>publication, </b>Journal<span class="s12"> </span>of<span class="s12"> </span>Biological<span class="s12"> </span>Chemistry<span class="s8"><b><span class="Apple-converted-space"> </span></b></span></p>
<p class="p24">“Competition<span class="s12"> </span>between<span class="s27"> </span>two high- and low-affinity protein-binding sites in myosin VI controls its cellular<span class="s22"> </span>function”</p>
<p class="p22"><span class="Apple-tab-span">	</span><span class="Apple-tab-span">	</span><span class="s8"><b><i>Published</i></b></span></p>
<p class="p23"><i></i><br></p>
<p class="p23"><i></i><br></p>
<p class="p30"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1"></span><br></p>
</body>
</html>
